userinterface/vehicleparameterui.ui

```diff
@@ -171,6 +174,1986 @@
        <string>20</string>
       </property>
      </row>
+     <row/>
```

Comment:
Would it not be better to setRowCount programmatically to correct size when you receive the parameters?
https://doc.qt.io/qt-5/qtablewidget.html#setRowCount

Reply:
Agree, will add that instead